fix: removed ollama embeddings. fastembeddings or openai only

o-on-x opened the pull request.

monilpat commented:
Thanks for getting this out - LGTM! If there is anyway to add more details in the description as to why it is needed that would be amazing. Otherwise it seems pretty innocuous !

sirkitree commented:
@o-on-x I too would love to hear why this is needed, out of my own curiosity :)

o-on-x commented:
@sirkitree @monilpat i had added this as way to use ollama's embedding endpoints. issue is this switches the model loaded in ollama each time an embedding is needed. at the time of, there was no good local embedding solution. now there is fastembedding. so it makes sense to just have either fastembedding or openai. also this resolves a breaking issue with anyone using ollama on newest codebase. i had added this code, i was removing it now that a better solution exists  https://github.com/ai16z/eliza/pull/221/files
